src/test/java/ibm/jceplus/junit/openjceplusfips/multithread/TestHmacSHA3_224.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright IBM Corp. 2023
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+
+package ibm.jceplus.junit.openjceplusfips.multithread;
+
+
+import ibm.jceplus.junit.base.BaseTestHmacSHA3_224;
+import ibm.jceplus.junit.openjceplus.Utils;
+
+public class TestHmacSHA3_224 extends ibm.jceplus.junit.base.BaseTestHmacSHA3_224 {
+
+    //--------------------------------------------------------------------------
+    //
+    //
```

Comment:
For new tests could we remove the empty comments similar to this one?

Reply:
Removed.